atlasdb-config/src/main/java/com/palantir/atlasdb/factory/Leaders.java

```diff
@@ -206,12 +209,31 @@ public static LocalPaxosServices createInstrumentedLocalServices(
         List<PingableLeader> remotePingableLeaders = otherLeaders.stream()
                 .map(LeaderPingerContext::pinger)
                 .collect(Collectors.toList());
+
+        //todo Sudiksha
+        List<TimeLockCorruptionPinger> remoteCorruptionPingers = remotePaxosServerSpec.remoteAcceptorUris().stream()
```

Comment:
While this will work, it's wonky: you're using the acceptor URIs for a non-Acceptor resource.